contributing/DEVELOPING.md

```diff
@@ -42,11 +42,14 @@ The Quick Start Guide's  [One-time setup](getting-started-quick.md#one-time-setu
 | `gulp lint --watch`                                                     | Watches for changes in files, Validates against Google Closure Linter.|
 | `gulp lint --fix`                                                       | Fixes simple lint warnings/errors automatically.                      |
 | `gulp build`<sup>[[1]](#footnote-1)</sup>                               | Builds the AMP library.                                               |
+| `gulp build --extensions=<extension-name,extension-name>`               | Builds the AMP library with only listed extension components.
+| `gulp build --noextensions`                                             | Builds the AMP library with no extensions.
```

Comment:
Add these variations to `gulp` as well (above).

Reply:
done